src/Gateway/PayplugGateway.php

```diff
@@ -318,6 +317,12 @@ public function init_settings()
      */
     public function init_form_fields()
     {
+
+		//add fields of IP to the description
+		if($this->payment_method === 'integrated'){
+			$this->description = IntegratedPayment::template_form();
```

Comment:
`IntegratedPayment::template_form();` should be inside `payment_fields` also the property of the gateway `has_fields` should be turned to true in the constructor when the payment method is Integrated. 

Like that the form will be shown on the write place on the html code of the payment methods on the checkout page.